tests/test_news.py

```diff
@@ -57,13 +57,14 @@ def test_date_only(self):
         'action': 'opened',
         'number': 1234,
         'pull_request': {
+            'url': 'https://api.github.com/repos/cpython/python/pulls/1234',
             'statuses_url': 'https://api.github.com/some/status',
             'issue_url': 'https://api.github.com/repos/cpython/python/issue/1234',
         },
     }
     event = sansio.Event(event_data, event='pull_request', delivery_id=1)
-    await news.router.dispatch(event, gh)
-    assert gh.getiter_url == '/repos/python/cpython/pulls/1234/files'
+    await news.check_news(gh, event_data['pull_request'])
+    assert gh.getiter_url == 'https://api.github.com/repos/cpython/python/pulls/1234/files'
```

Comment:
Any reason for adding the domain?

Reply:
When I ran `pytest`, the test was failing the other way.  I think it was because of how I changed the iteration.  I wanted to pull the repo name from the JSON instead of hard-coding it for when I was trying to run this over my own repo during testing.  `news.py` originally had this line:
``` python
files_url = f'/repos/python/cpython/pulls/{pr_number}/files'
```
which I made as the following in `util.py`
``` python
    files_url = f'{pull_request["url"]}/files'
```